daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -160,6 +160,8 @@ runNew targetFolder templateName = do
         writeFileUTF8 configPath config
         removeFile configTemplatePath
 
+    putStrLn $ "Created a new project in \"" <> targetFolder "\"."
```

Comment:
Would be nice to include the template name in the note as well.

Reply:
Great idea, added!